pages/interface/components/DialogTopics/index.js

```diff
@@ -0,0 +1,58 @@
+import { Box, Dialog } from '@primer/react';
+import { useEffect, useRef, useState } from 'react';
+
+import { Link } from '@/Link';
```

Comment:
No projeto temos um padrão para importar elementos visuais pelo `@/TabNewsUI`, isso facilita a substituição quando necessário. Então, ficaria assim:

```suggestion
import { useEffect, useRef, useState } from 'react';

import { Box, Dialog, Link } from '@/TabNewsUI';
```

E se você está usando um componente novo que ainda não é exportado pelo `TabNewsUI/index.js`, como é o caso do `Dialog`, basta adicionar a exportação lá (está em ordem alfabética).